validator/validator-main.protoascii

```diff
@@ -1494,7 +1494,29 @@ tags: {
     value_regex: "[0-9]*"
   }
 }
-# 4.4.8 The dl element
+# 4.11.1 (HTML5.3) The details element
+tags: {
+  html_format: AMP
+  html_format: AMP4ADS
+  html_format: AMP4EMAIL
+  html_format: EXPERIMENTAL
+  tag_name: "DETAILS"
+  attrs: {
+    name: "open"
+    value: ""
+  }
+  attrs: { name: "[open]" }
+}
+# 4.11.2 (HTML5.3) The summary element
+tags: {
+  html_format: AMP
+  html_format: AMP4ADS
+  html_format: AMP4EMAIL
+  html_format: EXPERIMENTAL
+  tag_name: "SUMMARY"
+  mandatory_parent: "DETAILS"
+}
+# 4.4.9 The dl element
```

Comment:
Please leave as 4.4.8 as these are following the HTML5 spec.

Reply:
Fixed in 7255acd0